src/NHibernate/Mapping/OneToMany.cs

```diff
@@ -23,7 +23,7 @@ public OneToMany(PersistentClass owner)
 
 		private EntityType EntityType
 		{
-			get { return TypeFactory.ManyToOne(ReferencedEntityName, null, false, false, IsIgnoreNotFound, false); }
+			get { return TypeFactory.ManyToOne(ReferencedEntityName, null, false, false, IsIgnoreNotFound, false, null); }
```

Comment:
This strange. Why `OneToMany` mapping returns `ManyToOne` entity type? If it's not a bug we need to add explcit comment about it. Looks very confusing.